notifications_utils/formatters.py

```diff
@@ -134,6 +136,10 @@ def strip_dvla_markup(value):
     return re.sub(dvla_markup_tags, '', value)
 
 
+def url_encode_full_stops(value):
+    return full_stops.sub('%2E', value)
```

Comment:
You could do `value.replace('.', '%2E')` here to make it a bit simpler since we're not going to add anything else we encode to '%2E' to that regex